### DISCUSSION TITLE
Ctrl+Enter

### DISCUSSION THREAD

**Discussion by NilsGN** [jendrikseipp/rednotebook#746] [General]
Is it really necessary?
To keep the text readable, I have to add a lot of Ctrl-Enter.
Can I fix it on my own?
Regards

**Comment by jendrikseipp** [comment]
Ctrl-Enter is needed if you want a line break without starting a new paragraph. (A new paragraph is started by inserting one empty line.) That's a decision made by the underlying txt2tags markup language.

**Reply by NilsGN** [reply]
OK. Thank you very much for your reply :)
